### PR TITLE
feature/four-9579: Action: Open the Screen and expand to cover about 50% of the screen

### PR DESCRIPTION
**Pull request by fagubla**
## Issue & Reproduction Steps
Task Preview: Improve the PRD request related to
Action: Open the Screen and expand to cover about 50% of the screen

## Solution
Create a new view, I duplicated the view edit of tasks so I remove the code useless for this new view

## How to Test
npm run dev
php artisan route:clear

## Related Tickets & Packages
- [FOUR-9579](https://processmaker.atlassian.net/browse/FOUR-9579)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

## Enviroment
ci:deploy

[FOUR-9579]: https://processmaker.atlassian.net/browse/FOUR-9579?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-970f0ea7e5.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-970f0ea7e5.eng.processmaker.net

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: ProcessMaker/Http/Controllers/TaskController.php [hunk 1]
```diff
@@ -73,6 +73,17 @@ public function edit(ProcessRequestToken $task)
         if ($element instanceof ScriptTaskInterface) {
             return redirect(route('requests.show', ['request' => $task->processRequest->getKey()]));
         } else {
+            if ($preview !== '') {
```
Comment (pmPaulis) [inline ProcessMaker/Http/Controllers/TaskController.php:76]: please change to:
`if (!empty($preview)) {`

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-970f0ea7e5.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-970f0ea7e5.eng.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-970f0ea7e5.eng.processmaker.net

**Pull request merged**